.github/ISSUE_TEMPLATE/bug.yml

```
@@ -17,6 +17,17 @@ body:
       placeholder: e.g. Expressions with empty object literals cause an error.
     validations:
       required: true
+  - type: input
+    id: reproduction-url
+    attributes:
+      label: Link to Minimal Reproducible Example
+      description: |
+        Create a minimal reproduction of the problem. **A minimal reproduction is required** so that others can help debug your issue. If a report is vague (e.g. just a generic error message) and has no reproduction, it may be closed.
```

[suggest] How about emphasizing a reproduction URL like this, though it seems a bit too long? 🤔 

```suggestion
        Provide a minimal reproduction URL like a GitHub repository or a StackBlitz project. **A minimal reproduction is required** so that others can help debug your issue. If a report is vague (e.g. just a generic error message) and has no reproduction, it may be closed.
```

Seems better if a shorter description would be good (one sentence if possible), but I don't have any good idea... 😓 (feel free to ignore this suggestion)

---

@jeddy3 Any ideas?

---

Off topic, but I'm afraid StackBlitz can't run vscode extensions, unlike GitHub Codespaces.

---

I see. Indeed, StackBlitz [doesn't support 3rd-party extensions](https://github.com/stackblitz/core/issues/3).

It seems GitHub Codespaces [deprecates public access](https://docs.github.com/en/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces#access-and-security), so a GitHub repository is most suitable as a reproduction environment.